### ISSUE TITLE
Document what user should install in their home

### ISSUE THREAD

**Issue by tcompa** [fractal-analytics-platform/fractal-server#120]
After #112 and its follow-ups (e.g. #114) reach some stability, let's make sure that we document the steps the user should take. This could include, for instance, creating an environment in their home and installing the fractal packages in it.

**Comment by tcompa** [comment]
Closing in favor of updating https://github.com/fractal-analytics-platform/fractal-demos/tree/main/examples.
